chart/kubeapps/values.yaml

```diff
@@ -542,19 +542,22 @@ dashboard:
 ## ref: https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml
 ##
 postgresql:
+  # Enable PostgreSQL. This should be true
+  enabled: true
   ## Enable replication for high availability
   replication:
     enabled: true
   ## Create a database for Kubeapps on the first run
   postgresqlDatabase: assets
+  ## PostgreSQL admin password (used when `postgresqlUsername` is not `postgres`)
+  ## ref: https://github.com/bitnami/bitnami-docker-postgresql/blob/master/README.md#creating-a-database-user-on-first-run (see note!)
+  ##
+  postgresqlPassword: ""
```

Comment:
Can we do:
```
postgresqlPassword:
```
instead? (ie. not explicitly put empty quotes, as even though it's functionally the same, it looks like we're overriding it to be an empty string, rather than leaving it unset as it is in the postgresql chart)